mapstr/mapstr_test.go

```diff
@@ -1264,6 +1264,22 @@ func TestAlterPath(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:      "alters keys on second level with case-insensitive matching",
```

Comment:
I think this what this case is testing, right?

```suggestion
			name:      "does not return an error if alterFunc returns the key unchanged",
```